doc/whats_new.rst

```diff
@@ -28,11 +28,21 @@ v0.8.1) will document the latest features.
 * Deprecate the ``is_stationary`` method in tests of stationarity. This will be removed in
   v1.4.0. Use ``should_diff`` instead.
 
-* Added two new datasets: ``airpassengers`` & ``austres``
+* Add two new datasets: ``airpassengers`` & ``austres``
 
```

Comment:
I assume we're trying to keep the "What's new" page in present tense? _Very_ nit-picky, but some of these are past tense (Added the `OCSBTest` of seasonality, as discussed in #88), and some are present tense

Reply:
Yeah we may even want it to be present progressive, like "Version 1.2.0 *adds* <X feature>"